app/addons/documents/shared-resources.js

```diff
@@ -27,7 +27,7 @@ define([
       return FauxtonAPI.constants.DOC_URLS.GENERAL;
     },
     url: function (context) {
-      if (context === undefined) {
+      if (!context) {
```

Comment:
i think the most cleanest way would be to remove the lazy context choosing, i grepped the code and just saw two occurences.

that way you have to pass the argument every time explicitly and the error logging in the url registry works in a reliable way